index.js

```diff
@@ -209,6 +213,47 @@ app.get("/popup", function (req, res) {
   res.status(200).send(popupMsg);
 });
 
+const proxyMiddleware = (target, pathRewrite) =>
+  createProxyMiddleware({
+    target,
+    changeOrigin: true,
+    pathRewrite,
+    selfHandleResponse: true,
+    onProxyRes: responseInterceptor(async (responseBuffer, proxyRes) => {
+      if (!proxyRes.headers["content-type"].includes("application/json"))
```

Comment:
Should be `?.includes(...)`, as this would fail if there's no `Content-Type` header in the response.